native/cocos/core/Root.cpp

```diff
@@ -82,22 +119,14 @@ void Root::initialize(gfx::Swapchain *swapchain) {
     depthStencilAttachment.depthStoreOp = gfx::StoreOp::DISCARD;
     depthStencilAttachment.stencilStoreOp = gfx::StoreOp::DISCARD;
 
-    scene::IRenderWindowInfo info;
-    info.title = ccstd::string{"rootMainWindow"};
-    info.width = swapchain->getWidth();
-    info.height = swapchain->getHeight();
-    info.renderPassInfo = renderPassInfo;
-    info.swapchain = swapchain;
-    _mainWindow = createWindow(info);
-
-    _curWindow = _mainWindow;
+    scene::IRenderWindowInfo windowInfo;
+    windowInfo.title = StringUtil::format("renderWindow_%d", windowId);
```

Comment:
Use `renderWindow_X` for window title is not suitable.

Reply:
How to do better? But I think `RenderWindow` doesn't need a title.

Reply:
May be just `WindowX`, or can be passed outside?

Reply:
What's the difference between `WindowX` and `renderWindow_X`?

Reply:
As the title is for window, not for render window.